package.json

```diff
@@ -19,6 +19,7 @@
     "prettier:all": "node ./scripts/prettier.js write",
     "size:snapshot": "node --max-old-space-size=2048 ./scripts/sizeSnapshot/create",
     "size:why": "yarn size:snapshot --analyze --accurateBundles",
+    "t": "node test/cli.js",
```

Comment:
What does t do? Can we rename?

Reply:
This is described in the pull request description: https://github.com/mui-org/material-ui/pull/23369

Reply:
@eps1lon Maybe we should document it in https://github.com/mui-org/material-ui/pull/23369?

Reply:
Since this command is experimental and I haven't received any feedback, I'm hesitant to document it.

Reply:
What would be the criteria for promoting the command from the experimental state? When it comes to running tests in jsdom, I'm almost exclusively using it (over `yarn test:watch`).